CHANGELOG.md

```diff
@@ -2,6 +2,12 @@ Note: This is in reverse chronological order, so newer entries are added to the
 
 Swift 5.6
 -----------
+
+* [#3890]
```

Comment:
need to also add the link at the bottom, e.g. https://github.com/apple/swift-package-manager/pull/3988/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL171

Reply:
https://github.com/apple/swift-package-manager/pull/3988/commits/f6b08903510744a897260684514b3e37f22ce188